src/main/scala/se/nimsa/sbx/app/routing/SeriesTypeRoutes.scala

```diff
@@ -138,7 +139,19 @@ trait SeriesTypeRoutes { this: SliceboxBase =>
             }
           }
         }
-
+      } ~ pathPrefix("series") {
+        path("query") {
+          pathEndOrSingleSlash {
```

Comment:
The `pathPrefix` then `path` then `pathEndOrSingleSlash` can be contracted to a single line of `path("series" / "query")`. `path` is terminal so there is no need for a `pathEndOrSingleSlash` after a `path`, only after a `pathPrefix`.